dill/session.py

```diff
@@ -149,6 +223,12 @@ def dump_module(
             similar but independent from ``dill.settings[`byref`]``, as
             ``refimported`` refers to virtually all imported objects, while
             ``byref`` only affects select objects.
+        refonfail: if `True`, objects that fail to pickle by value will try to
+            be saved by reference.  If this also fails, saving their parent
+            objects by reference will be attempted recursively.  In the worst
+            case scenario, the module itself may be saved by reference.  Note:
+            The file-like object must be seekable and truncable with this
```

Comment:
truncatable

Reply:
I've never met this word before, in any language. I probably guessed it wrong with a Latin structure.